server/dist/src/main/resources/concord-server.conf

```diff
@@ -112,7 +112,9 @@ concord-server {
             # requires "CREATE EXTENSION" privileges
             createExtensionAvailable = "true"
 
-            secretStoreSalt = "${secretStore.secretStoreSalt}"
```

Comment:
i tried it, referencing the value from same file is not working. correct me if i am wrong. thats why added these values directly. @brig

Reply:
hm... interesting. is `secretStore.secretStoreSalt` is defined in your cfg? share error?

Reply:
I think the problem is `""` :(

Reply:
updated.